config.go

```diff
@@ -43,8 +43,8 @@ func ReadConfig(path string) (c Config, err error) {
 func readConfig(r io.Reader) (c Config, err error) {
 	// Unfortunately the YAML package does not
 	// support reader inputs
-  // TODO(aditya) convert this when the
-  // upstream PR lands
+	// TODO(aditya) convert this when the
+	// upstream PR lands
```

Comment:
nitpick: link to PR in question? (I should have nitted on this when the comment was added but clearly I forgot)